packages/shared-influx-pieces/src/lib/utils/schedule.ts

```diff
@@ -0,0 +1,21 @@
+import { TimerManager } from '@sapphire/timer-manager';
+import { container } from '@skyra/http-framework';
+import { getApproximateGuildCount } from './ApproximateGuilds.js';
+
+const Minute = 60;
```

Comment:
The time is in milliseconds.
```suggestion
const Minute = 60000;
```